### DISCUSSION TITLE
O que são GET e POST?

### DISCUSSION THREAD

**Discussion by ameninadogorro** [community/community#36457] [Apps, API and Webhooks] [answered]
O que seria esses dois métodos e pra que eles servem?

**Comment by pedrohgmuniz** [comment, marked as answer]
São métodos de HTTP, o GET é um método que solicita a representação de um recurso específico, requisições utilizando método GET devem retornar apenas dados. O POST é um método que é utilizado para submeter uma entidade a um recurso específico, frequentemente causando uma mudança no estado do recurso ou efeitos colaterais no servidor. Submeter algo ao servidor de forma mais discreta.

**Comment by ettaboyle** [comment]
Thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
